### PR TITLE
041 - implement video teaser into homepage

### PR DESCRIPTION
**Pull request by garySiladi**
This PR introduces the video teasers into the homepage. As the user selects an episode, there is a little blurred version of the video playing in the opened box.

### CODE REVIEW

File: src/components/episode-selected/episode-selected.js [hunk 1]
```diff
@@ -86,8 +90,14 @@ class episodeSelected extends React.Component {
...
-        <div className="episode-selected__image-wrapper">
-          <img className={imageClassName} src={url} alt={title} />
+        <div className="episode-selected__teaser-wrapper">
+          <VideoPlayer
+            videoUrl={videoUrl}
+            showUI={false}
+            posterImage={url}
+            isMuted
+          />
+          <img className={imageClassName} src={PlayLogo} alt={title} />
```
Comment (varholak-peter) [inline src/components/episode-selected/episode-selected.js]: This should be using the video UI component inside Video Player

File: src/components/video-player-container/parts/video-player.js [hunk 2]
```diff
@@ -14,15 +14,19 @@ require('videojs-contrib-hls/dist/videojs-contrib-hls.js');
...
+  isMuted: boolean,
+  posterImage: ?string,
 };
 export type VideoPlayerStateType = {
   isVideoPlaying: boolean,
   timeProgress: number,
 }
 
-const videoJsOptions = (videoUrl: string) => ({
+const videoJsOptions = (videoUrl: string, isMuted) => ({
```
Comment (varholak-peter) [inline src/components/video-player-container/parts/video-player.js]: we either don't need type for videoUrl or we need type for isMuted

File: src/components/video-player-container/parts/video-player.js [hunk 3]
```diff
@@ -52,17 +56,20 @@ class VideoPlayer extends React.Component {
...
   state: VideoPlayerStateType;
-
   componentDidMount() {
-    (this: any).player = videojs((this: any).videoNode, { ...videoJsOptions(this.props.videoUrl) });
+    const {
+      videoUrl,
+      isMuted,
+    } = this.props;
+    (this: any).player =
+      videojs((this: any).videoNode, { ...videoJsOptions(videoUrl, isMuted) });
```
Comment (varholak-peter) [inline src/components/video-player-container/parts/video-player.js]: can this not be in a single line as before?

File: src/components/episode-selected/episode-selected.js (hunk 1 above)
Comment (garySiladi) [inline src/components/episode-selected/episode-selected.js]: this isn't really a button as a control, just an image ...
also by wireframes this segment will be different

**Pull request merged**